index.html

```diff
@@ -12767,7 +12767,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p>This property is used, for example, on a range widget such as a slider or progress bar.</p>
 				<p>If the <pref>aria-valuetext</pref> attribute is set, authors SHOULD also set the <pref>aria-valuenow</pref> attribute, unless that value is unknown (for example, on an indeterminate <rref>progressbar</rref>).</p>
 				<p>Authors SHOULD only set the <pref>aria-valuetext</pref> attribute when the rendered value cannot be meaningfully represented as a number. For example, a slider may have rendered values of <code>small</code>, <code>medium</code>, and <code>large</code>. In this case, the values of <pref>aria-valuenow</pref> could range from 1 through 3, which indicate the position of each value in the value space, but the <pref>aria-valuetext</pref> would be one of the strings: <code>small</code>, <code>medium</code>, or <code>large</code>. If the <pref>aria-valuetext</pref> attribute is absent, the <a>assistive technologies</a> will rely solely on the <pref>aria-valuenow</pref> attribute for the current value.</p>
-				<p>If aria-valuetext is specified, assistive technologies SHOULD render that value instead of the value of <rref>aria-valuenow</rref>.</p>
+				<p>If aria-valuetext is specified, assistive technologies SHOULD render that value instead of the value of <pref>aria-valuenow</pref>.</p>
```

Comment:
```suggestion
				<p>If <pref>aria-valuetext</pref> is specified, assistive technologies SHOULD render that value instead of the value of <pref>aria-valuenow</pref>.</p>
```